sagemathinc/cocalc
# Typescript a Few Things

# Description
Typescript Tip, Loading, and Space
Typescript feature.coffee


# Testing Steps
1. Use Browserstack to or Chome dev tools to confirm platform detection works
1. Checked that Tips, Loading, and Space work in Project Files and Courses.

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):
